src/helm.exec.ts

```diff
@@ -173,6 +173,26 @@ export function helmGet(resourceNode: explorer.ResourceNode) {
     });
 }
 
+// helmPackage runs the Helm package on a chart within your project.
+export function helmPackage() {
+    pickChart((path) => {
+        vscode.window.showInputBox({
+            prompt: "package path",
+            placeHolder: "package path"
+        }).then((ppath) => {
+            const packagepath = ppath;
```

Comment:
I think you need to check `if (ppath)` - if the user cancels `showInputBox` then the continuation is still called but with an `undefined` argument.